docs.go

```diff
@@ -2,7 +2,7 @@
 Package flagsfiller makes Go's flag package pleasant to use by mapping the fields of a given struct
 into flags in a FlagSet.
 
-Quick Start
+# Quick Start
```

Comment:
Are you sure markdown syntax works here?

The doc rendering already is aware of section headers:

![image](https://user-images.githubusercontent.com/988985/236645817-062a18db-1dde-4947-a14a-794fa0ce3649.png)

Reply:
My apologies, I didn't know they have since added markdown support:

https://tip.golang.org/doc/comment#syntax